### PR TITLE
Minor changes to Formatter logic

### PR DESCRIPTION
**Pull request by jjatie**

### CODE REVIEW

File: Source/Charts/Formatters/DefaultAxisValueFormatter.swift [hunk 1]
```diff
@@ -90,14 +91,7 @@ open class DefaultAxisValueFormatter: NSObject, IAxisValueFormatter
...
     {
-        if block != nil
-        {
-            return block!(value, axis)
-        }
-        else
-        {
-            return formatter?.string(from: NSNumber(floatLiteral: value)) ?? ""
-        }
+        return block?(value, axis) ??
```
Comment (petester42) [inline Source/Charts/Formatters/DefaultAxisValueFormatter.swift]: I think `if let` would be more readable in this case.

File: Source/Charts/Formatters/DefaultValueFormatter.swift [hunk 2]
```diff
@@ -94,14 +94,7 @@ open class DefaultValueFormatter: NSObject, IValueFormatter
...
     {
-        if block != nil
-        {
-            return block!(value, entry, dataSetIndex, viewPortHandler)
-        }
-        else
-        {
-            return formatter?.string(from: NSNumber(floatLiteral: value)) ?? ""
-        }
+        return block?(value, entry, dataSetIndex, viewPortHandler) ??
```
Comment (petester42) [inline Source/Charts/Formatters/DefaultValueFormatter.swift]: I think `if let` would be more readable in this case.

Comment (jjatie) [reply Source/Charts/Formatters/DefaultValueFormatter.swift]: I flip-flopped on this one a lot. I'll change it now.

**Comment by jjatie** [comment]
Changes have been made

**Comment by jjatie** [comment]
@petester42 Any further feedback on this one or can it be merged?

**Comment by danielgindi** [comment]
Actually the naming of the commit / PR itself could have been better, as it does not change the logic - it's a refactoring :-)

**Comment by liuxuan30** [comment]
and I already told him to try squash before merging and our idea for managing commits

**Pull request merged**